src/lib/dom_helpers.js

```diff
@@ -32,7 +43,7 @@ function bindFocusables( instance, activateOnFocus ) {
             if ( onFocusPrev ) onFocusPrev.call( element, event );
           };
         };
-        for ( let element of [ ...focusables ] ) {
+        for ( let element of nodeListToArray(focusables) ) {
           element.onfocus = onFocus( element );
         }
```

Comment:
Right, yeah I expect problems on a few different fronts for IE < 9. Let's go with slice and I'll make it clear in the readme that we're not supporting IE8. Thanks again!